package.json

```diff
@@ -15,7 +15,7 @@
     "lint": "prettier --plugin prettier-plugin-svelte . --check . && eslint .",
     "format": "prettier --plugin prettier-plugin-svelte . --write .",
     "docgen-props": "NODE_OPTIONS='--loader ts-node/esm/transpile-only --no-warnings' node scripts/docgenProps.ts",
-    "colorgen": "NODE_OPTIONS='--loader ts-node/esm/transpile-only --no-warnings' node ./src/dev/colorgenPlugin.ts",
+    "colorgen": "node --loader ts-node/esm/transpile-only --no-warnings scripts/colorgen.ts",
```

Comment:
Why did you change the `NODE_OPTIONS`? Does it bring compatibility issues with certain node versions? Also, we should (later) try to replace `ts-node` with `tsx`.

Reply:
This doesn't work on Windows, it says that `NODE_OPTIONS is not recognised as an internal or external command`